test/gurka/test_conditional_restrictions.cc

```diff
@@ -174,3 +179,60 @@ TEST_F(ConditionalRestrictions, RestrictionPedestrian) {
       },
       std::exception);
 }
+
+TEST_F(ConditionalRestrictions, DestinationRestrictionOnLastEdgeIsValid) {
+  for (auto const& date_time_type : kDateTimeTypes) {
+    auto result = gurka::do_action(valhalla::Options::route, map, {"F", "I"}, "auto",
```

Comment:
Checks for other costings (motorcycle, tuck, etc) as long as it is implemented for them?

Reply:
Done